src/components/CommandPalette.tsx

```diff
@@ -291,6 +313,10 @@ const CommandPalette: FC = () => {
 
   const [selectedShortcut, setSelectedShortcut] = useState<Command>(shortcuts[0])
 
+  // Get help command info for gesture matching
+  const helpCommand = useMemo(() => globalCommands.find((cmd: Command) => cmd.id === 'help'), [])
```

Comment:
We have [commandById](https://github.com/cybersemics/em/blob/75b8c516e70922bdfa44d29a8cc47e33c942e865/src/commands.ts#L325-L326) which operates in O(1) rather than O(n).

This can either be inlined below or taken completely outside the component and put at the module level. We shouldn't have to store or memoize it at the component level since it is not connected to the component's lifecycle.